src/Symfony/Bridge/Twig/Resources/views/Form/foundation_5_layout.html.twig

```diff
@@ -165,21 +165,25 @@
 {%- endblock choice_widget_collapsed %}
 
 {% block choice_widget_expanded -%}
-    {% if '-inline' in label_attr.class|default('') %}
+    {% if '-inline' in label_attr.class|default('') -%}
```

Comment:
@jakzal ok reverted. `foundation_5_layout` was introduced on 2.8 should I make a PR on this branch or on master ? Do you think it's really needed ?